docs/build-release/tasks/utility/download-secure-file.md

```diff
@@ -20,6 +20,8 @@ monikerRange: 'vsts'
 
 Use this task to download a [secure file](../../concepts/library/secure-files.md) from the server during a build or release.
 
```

Comment:
Can you also add a note that the full path to the downloaded file is accessible via $env:DOWNLOADSECUREFILE_SECUREFILEPATH. If multiple versions of the task are used the variables are disambiguated as DOWNLOADSECUREFILE1_SECUREFILEPATH, DOWNLOADSECUREFILE2_SECUREFILEPATH ...

Reply:
Sure, I can do that!